spec/reversearray.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function reversearray( array ) {
+  return array.reverse()
```

Comment:
on this project the idea is to reverse engineer the function...so it defeats the purpose if you use ```reverse()``` for ```reverse``` 😏